apps/andi/lib/andi_web/controllers/reports_controller.ex

```diff
@@ -0,0 +1,14 @@
+defmodule AndiWeb.ReportsController do
+  use AndiWeb, :controller
+
+  access_levels(download_report: [:private])
+
+  def download_report(conn, _params) do
```

Comment:
super minor, but we can probably just remove _params if we're not using it unless it's going to be used in a future iteration and is just a placeholder

Reply:
because controllers are called through phoenix there is actually a blank params value sent to the function. So removing it will cause an error saying there isn't a function match with arity 2.